docs/reference/edot-java/supported-technologies.md

```diff
@@ -17,6 +17,21 @@ products:
 
 The EDOT Java agent is a [distribution](https://opentelemetry.io/docs/concepts/distributions/) of [OpenTelemetry Java Instrumentation](https://github.com/open-telemetry/opentelemetry-java-instrumentation) agent. It inherits all the [supported](opentelemetry://reference/compatibility/nomenclature.md) technologies of the OpenTelemetry Java Instrumentation.
 
+:::{note}
+**Understanding auto-instrumentation scope**
+
+Auto-instrumentation automatically captures telemetry for the frameworks and libraries listed on this page. However, it cannot instrument:
+
+- Custom or proprietary frameworks and libraries
+- Closed-source components without instrumentation support
+- Application-specific business logic
+
+If your application uses technologies not covered by auto-instrumentation, you have two options:
+
+1. **Native OpenTelemetry support** — Some frameworks and libraries include built-in OpenTelemetry instrumentation provided by the vendor.
+2. **Manual instrumentation** — Use the [OpenTelemetry API](https://opentelemetry.io/docs/languages/java/instrumentation/) to add custom spans, metrics, and logs for unsupported components.
```

Comment:
You could maybe add a third option is to use manual instrumentation through configuration, it does not require to modify application code or dependencies: https://opentelemetry.io/docs/zero-code/java/agent/annotations/#creating-spans-around-methods-with-otelinstrumentationmethodsinclude

We already mention this option [in our migration docs](https://www.elastic.co/docs/reference/opentelemetry/edot-sdks/java/migration#trace_methods) for the `trace_methods` config option of the classic agent as it serves a similar purpose.